import System.Exit
main = exitWith (ExitFailure 42)
